src/vendor/jx/expressions.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-use-before-define */
```

Comment:
Mutually recursive functions:  Specifically, the poorly named  `jx` function uses the `expressions` object and vice-versa.  I find the code is more elegant if each is defined once, as a declaration, rather than using procedural composition.  

Since `no-use-before-define` raises a question, I will add more code so I can remove the disable.

Reply:
I think an explanatory comment indicating that `jx` and `expressions` are mutually recursive is probably sufficient here; I'll admit that I missed that fact on my first pass.  (This is one of those cases where I think I agree with you rather than with the linter, but I do think the explanatory comment would help anyone reading the code.)